### PR TITLE
Docs: fix grammar issues

### PR DESCRIPTION
**Pull request by suojy**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Fixes typos in docs

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: docs/components/edgesite.md [hunk 1]
```diff
@@ -10,15 +10,15 @@ EdgeSite helps running lightweight clusters at edge.
 
 ## Motivation
 
-There are scenarios user need to run a standalone Kubernetes cluster at edge to get full control and improve the offline scheduling capability. There are two scenarios user need to do that:
+There are scenarios user needs to run a standalone Kubernetes cluster at edge to get full control and improve the offline scheduling capability. There are two scenarios user need to do that:
```
Comment (fisherxu) [inline docs/components/edgesite.md:13]: users need?

**Comment by daixiang0** [comment]
Better title: "Docs: fix gramma issues"

**Comment by fisherxu** [comment]
@suojy Would you like to squash your commits? :)

**Comment by suojy** [comment]
> @suojy Would you like to squash your commits? :)

Yes, but I don't know how to squash commits from my branch. Could you help me with that？Thanks.

**Comment by daixiang0** [comment]
Run `git rebase` command, for more details you can google it or man.

**Comment by fisherxu** [comment]
You can ref to this blog: https://medium.com/@slamflipstrom/a-beginners-guide-to-squashing-commits-with-git-rebase-8185cf6e62ec  @suojy 

```
git rebase -i <The Commit ID after your first commit>

use `s` or `f` to squash the commits.

pick b608d9f fixes typos in docs
f 050b968 Update edgesite.md

```

**Comment by daixiang0** [comment]
ping @suojy

**Comment by suojy** [comment]
> ping @suojy

Sorry, I didn't get it. Do you mean that I should run ping github.com? It works.

**Comment by daixiang0** [comment]
@suojy just like a tip :)

**Comment by suojy** [comment]
> @suojy just like a tip :)

thanks 😂

**Comment by suojy** [comment]
> @suojy just like a tip :)

I ran git rebase and just wonder if I squashed the commits correctly.

**Comment by fisherxu** [comment]
@suojy  I will `squash merge` the PR manually. But I'd suggest you should still explore how to squash the commits :)

**Pull request merged**